src/game/AI/EventAI/CreatureEventAI.cpp

```diff
@@ -167,7 +167,7 @@ void CreatureEventAI::InitAI()
                 {
                     m_CreatureEventAIList.push_back(CreatureEventAIHolder(aiEvent));
                     // Cache for fast use
-                    if (aiEvent.event_type == EVENT_T_OOC_LOS)
+                    if (aiEvent.event_type == EVENT_T_OOC_LOS || EVENT_T_OOC_NO_LOS)
```

Comment:
I doubt this does what you intended, because it's missing the ```aiEvent.event_type ==``` part after the logical or.